DevelopmentWorkflow.md

```diff
@@ -0,0 +1,68 @@
+# Development Workflow
+
+<!-- TOC depthFrom:2 -->
+
+- [Prerequisites](#prerequisites)
+- [Develop locally](#develop-locally)
+- [Build the plugin](#build-the-plugin)
+- [Testing the plugin](#testing-the-plugin)
+
+<!-- /TOC -->
+
+## Prerequisites
+
+- Install your native toolchain and NativeScript as [described in the docs](https://docs.nativescript.org/start/quick-setup)
+
+- Review [NativeScript plugins documentation](https://docs.nativescript.org/plugins/plugins) for more details on plugins development
+
+## Develop locally
+
+For local development we recommend using the npm commands provided in the plugin's package.json
+
+To run and develop using TypeScript demo:
+
+```bash
+# Go to demo directory
+cd nativescript-camera/demo
+# Build the plugin. This compiles all TypeScript files in the plugin directory, which is referenced in the demo's package.json
+npm run build.plugin
+# Install demo dependencies
+npm i
+# Run the demo for iOS or Android.
+tns run ios
+tns run android
+```
+
+After all the changes are done make sure to test them in all the demo app.
+
+Available npm commands from the plugin source directory:
+
+- `build` - builds the plugin so it can be used in the demo app.
+- `tslint` - check plugin's TypeScript sources for linting errors.
+- `clean` - clean the demo and plugin directories and build the plugin.

```

Comment:
I would expect the clean command to clean only, and not build. A rebuild command should do clean + build. Maybe we can discuss on that.